lighthouse-cli/test/smokehouse/readme.md

```diff
@@ -0,0 +1,121 @@
+
+## Options
+
+See [`SmokehouseOptions`](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-cli/test/smokehouse/smokehouse.js#L23).
+
+## Test definitions
+
+| Name | Type | Description |
+| -- | -- | -- |
+| `id` | `string` | The string identifier of the test. |
+| `expectations` | `{lhr: Object, artifacts: Object}` | See below. |
+| `config` | `LH.Config.Json` (optional) | An optional Lighthouse config. If not specified, the default config is used.|
+| `runSerially` | `boolean` (optional) | An optional flag. If set to true, the test won't be run in parallel to other tests. Useful if the test is performance sensitive. |
+
+### Expectations
+
+The smoke test expectations can assert the values of the Lighthouse result (the `lhr`) and gathered `artifacts` for multiple URLs. The URL to be tested is specified in the expectations's `requestedUrl` field.
+
+The expectations are asserted as a subset of the actual results: all values in the expectectations must be in the actual results, but not all actual results must be asserted.
+
+Examples can be found in the [core tests](./test-definitions/).
+
+### Special numeric expectations
+
+If checking a number somewhere in the Lighthouse results, numeric comparisons can be used in place of a raw expected number. This allows asserting ranges or categories of numbers where the exact value isn't necessarily important, or to allow for expected variability in a test.
+
+The comparator is specified with a string, and the actual value being tested must be a number. Whitespace may be included in the string for readability.
+
+The following operators are supported:
+
+| Operator | Example|
+| -- | -- |
+| `>` | `'>0'` |
+| `>=` | `'>=5'` |
+| `<` | `'<1'` |
+| `<=` | `'<=10'` |
+| `+/-` | `'100+/-10'` |
+| `±` | `'100±10'` |
+
+### Special string expectations
+
+If checking a string somewhere in the Lighthouse results, a regular expression can be used in place of a string literal.
+
+## Pipeline
+
+The different frontends launch smokehouse with a set of tests to run. Smokehouse then coordinates the tests using a particular method of running Lighthouse (CLI, as a bundle, etc).
+
+```
+Frontends                                        Lighthouse Runners
++------------+
+|            |
+|  bin (CLI) +----+                                +--------------+
+|            |    |                                |              |
++------------+    |                            +-->+lighthouse+cli|
+                  |                            |   |              |
++------------+    |      +---------------+     |   +--------------+
+|            |    |      |               |     |
+|   node     +---------->+ smokehouse.js +-----+
+|            |    |      |               |     |   +--------------+
++------------+    |      +-------+-------+     |   |              |
+                  |              ^             +-->+  bundled+LH  |
```

Comment:
is `bundledLhRunner` the only other candidate for the `-bin` runner or are our other hopes for smokehouse/firehouse also going to be runners